lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.SpaceType.rb

```diff
@@ -614,6 +650,7 @@ def has_multi_lpd_values_user_data(user_data, space_type)
   # The sum of each space fraction in the user_data is assumed to be 1.0
   # @param user_data [Hash] user data from the user csv
   # @param space [OpenStudio::Model::Space]
+  # @return space_lighting_per_area [Float]
```

Comment:
syntax is @return [Float] value units. run `bundle exec rake doc` and see errors